Goal goal
---
bool success
bool configuration_change